middleware/service_broker_rate_limiter.rb

```diff
@@ -3,6 +3,13 @@
 
 module CloudFoundry
   module Middleware
+    RateLimitEndpoint = Struct.new(:endpoint_pattern, :request_methods)
+
+    RATE_LIMITED_ENDPOINTS = [
+      RateLimitEndpoint.new(%r{\A/v2/(service_instances|service_credential_bindings|service_route_bindings)}, %w[PUT POST DELETE PATCH]),
```

Comment:
The v2 paths are not correct (there is no /v2/service_credential_bindings endpoint). See removed is_service_request method.

Reply:
Aah, jep